netty/build.gradle

```diff
@@ -33,9 +33,3 @@ test {
     // Allow testing Jetty ALPN in TlsTest
     jvmArgs "-javaagent:" + configurations.alpnagent.asPath
 }
-
-jmh {
```

Comment:
this caused an error during :grpc-netty:animalsnifferJmh task.
error is about "Failed to check signatures"

Reply:
Check whether "./gradlew :grpc-netty:jmh"  runs. You don't need to do a full benchmark, but if a benchmark starts, it is probably fine.

Reply:
it was broken (it maybe due to some hack in the previous version of PR). now it works as expected.